src/views/SubjectAllQuestions/Question/Question.tsx

```diff
@@ -24,6 +32,10 @@ interface Props {
     answerIndex: number,
     answerValue: boolean,
   ) => void;
+  learntButton?: {
```

Comment:
How about `learntButtonData`?
```suggestion
  learntButtonData?: {
```

`learntButton` would indicate for me at first sight that this prop is a react component instead of info based on which we should render the button.